src/Microsoft.Identity.Web/Resource/ITokenAuthorizationOptions.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+
+namespace Microsoft.Identity.Web
+{
+    /// <summary>
+    /// Properties related to authorization of access tokens in web APIs.
+    /// </summary>
+    public interface ITokenAuthorizationOptions
```

Comment:
can we consider passing in authorization policy.

Reply:
@yordan-msft : you mean only for ASP.NET Core?
The mechanism we propose here also works for ASP.NET classic (OWIN), and any other framework.

Reply:
as opposed to what? Is there a mechanism that does not work for other frameworks? Let's talk it over.

Reply:
What's the data structure for an authorization policy, @yordan-msft ?